src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -128,23 +158,8 @@ const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputProps> = (
       {error && <small>Error uploading: {error}</small>}
       <small>Or pick one from our Sparkle profile pics</small>
       <div className="default-avatars-container">
-        {isLoadingAvatars && <div>Loading...</div>}
-        {!isLoadingAvatars &&
-          defaultAvatars.map((avatar, index) => {
-            return (
-              <div
-                key={`${avatar}-${index}`}
-                className="profile-picture-preview-container"
-                onClick={() => uploadDefaultAvatar(avatar)}
-              >
-                <img
-                  src={`${avatar}`}
-                  className="profile-icon profile-picture-preview"
-                  alt="your profile"
-                />
-              </div>
-            );
-          })}
+        {isLoading && <div>Loading...</div>}
+        {!isLoading && avatarImages}
```

Comment:
```suggestion
        {isLoading ? <div>Loading...</div> : avatarImages}
```

Reply:
[OOS] Do we have some kind of spinner component in the project?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1409/commits/041df52e62699a3a610407e6d105970072a53ce2

Reply:
> [OOS] Do we have some kind of spinner component in the project?

Not that I remember seeing.. closest is possibly the sparkle animation.. but that's not ideal for this use case.